environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - usgs-astrogeology
   
 dependencies:
-  - ale =0.10.0,<1
```

Comment:
Make sure to add this update to the recipe/meta.yaml otherwise the dev release builds will fail.